src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/events/InboundEventsListener.kt

```diff
@@ -0,0 +1,53 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.service.events
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.PropertyNamingStrategies
+import com.fasterxml.jackson.databind.annotation.JsonNaming
+import com.fasterxml.jackson.module.kotlin.readValue
+import io.awspring.cloud.sqs.annotation.SqsListener
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+
+@Component
+class InboundEventsListener(
+  private val mapper: ObjectMapper,
+  private val inboundEventsService: InboundEventsService,
+) {
+  companion object {
+    private val log: Logger = LoggerFactory.getLogger(this::class.java)
+  }
+
+  init {
+    log.info("Event listener started.")
+  }
+
+  @SqsListener("education-and-work-plan", factory = "hmppsQueueContainerFactoryProxy")
+  internal fun onMessage(rawMessage: String) {
```

Comment:
I'm sure we can do better than this with proper types - all this passing strings around and messing about with objectmapper is making me cry 😢 

We had properly typed message listener methods in VCA ....